### ISSUE TITLE
How can I test orbit with latest js-ipfs?

### ISSUE THREAD

**Issue by daviddias** [orbitdb/orbit#206]

**Comment by diasdavid** [comment]
Would love to be able to make orbit, orbit-db and ipfs-log testable without too much configuration, as easy as:

```shell
> git clone git@github.com/ipfs/{orbit, orbit-db, ipfs-log} && \
> npm install && \
> npm link && \
> npm run {test, benchmark}:{electron, browser}:{go-ipfs, js-ipfs}
```

**Comment by aphelionz** [comment]
I believe this can be closed as we're supporting up to js-ipfs 0.36 and go-ipfs 0.4.22 on `latest`
